CONTRIBUTING.md

```diff
@@ -14,13 +14,16 @@ Add code owner per tool in [CODEOWNER](https://github.com/Azure/azure-sdk-tools/
 
 ## README
 
-Please add README.md file for every tool to illustrate:
+1. Please add README.md file for every tool to illustrate:
 * The purpose of the tool.
 * Prerequisites before use
 * How to use, test and maintain the tool locally and remotely. 
 * Better to include where the tool is being used.
 
-Example: [README](https://github.com/Azure/azure-sdk-tools/blob/main/tools/http-fault-injector/README.md) 
+Example: [README.md](https://github.com/Azure/azure-sdk-tools/blob/main/tools/http-fault-injector/README.md) 
+
+2. Please add tool index of the tool in root [README.md](https://github.com/Azure/azure-sdk-tools/blob/main/README.md).
```

Comment:
In my opinion, "please" sounds a little odd to use in technical documentation. No need for politeness in impersonal and imperative instructions.

Also, I'd direct link to the index and rephrase a little, as the instruction was confusing to me without having looked at the readme first:

```suggestion
2. Add tool details to the index in the root [README.md](https://github.com/Azure/azure-sdk-tools/blob/main/README.md).
https://github.com/Azure/azure-sdk-tools/blob/main/README.md#index
```